crml/staking/src/rewards/mod.rs

```diff
@@ -94,6 +98,14 @@ decl_storage! {
 		pub QueuedEraRewards get(fn queued_era_rewards): VecDeque<BalanceOf<T>>;
 		/// Hold the latest not processed payouts and the era when each is accrued
 		pub Payouts get(fn payouts): VecDeque<(T::AccountId, BalanceOf<T>, EraIndex)>;
+		/// The amount of new reward tokens that will be minted on every staking era in order to
+		/// approximate the inflation rate. We calculate the target inflation based on
+		/// T::CurrencyToReward::TotalIssuance() at the beginning of a fiscal era.
+		TargetInflationPerStakingEra get(fn target_inflation_per_staking_era): BalanceOf<T>;
+		/// The staking era index that specifies the start of the current fiscal era.
```

Comment:
would need to change this comment to reflect that it's only the most recently tracked one, might be easier to just update the value 😅

Reply:
Done